presidio-analyzer/tests/test_in_pan_recognizer.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from tests import assert_result
+from presidio_analyzer.predefined_recognizers import InPanRecognizer
+
+
+@pytest.fixture(scope="module")
+def recognizer():
+    return InPanRecognizer()
+
+
+@pytest.fixture(scope="module")
+def entities():
+    return ["IN_PAN"]
+
+
+@pytest.mark.parametrize(
+    "text, expected_len, expected_position, expected_score",
+    [
+        # fmt: off
+        ("AAASA1111R", 1, (0,10), 0.6) ,
+        ("ABCPD1234Z", 1, (0, 10), 0.85),
+        ("ABCND1234Z", 1, (0, 10), 0.6),
+        ("A1111DFSFS", 1, (0,10),0.05),
```

Comment:
Can you please add a few negative examples?

Reply:
hi @omri374 , can you please have a look at tests/data/context_sentences_test.txt (last 3 ) and let me know if those are sufficient.

Reply:
Thanks. Please see a reference here: https://github.com/microsoft/presidio/blob/1b470616ab422ca145f438e96af969e9f18646d1/presidio-analyzer/tests/test_ip_recognizer.py#L23

There are cases where the input should not be detected, and the expected length is 0. Furthermore, there are cases where the input is part of a sentence (to check that the patterns are not limiting to full text patterns only)